Source/Tester.js

```diff
@@ -17,126 +17,116 @@ import { Document } from './Document.js'
 */
 
 export class Tester {
-
-    /**
+  /**
      Make a new tester.
-
      */
 
-    constructor(application) {
-        /** @type {array} List of failures in the currently running test. */
-        this._testFailures = []
+  constructor(application) {
+    /** @type {array} List of failures in the currently running test. */
+    this._testFailures = []
 
-        /** @type {Application} The application that is running these tests. */
-        this._application = application
+    /** @type {Application} The application that is running these tests. */
+    this._application = application
 
-        /** @type {number} The number of tests we've run. */
-        this._ran = 0
+    /** @type {number} The number of tests we've run. */
+    this._ran = 0
 
-        /** @type {array} The names of the tests that have passed. */
-        this._passes = []
+    /** @type {array} The names of the tests that have passed. */
+    this._passes = []
 
-        /** @type {array} Failure information for each test that has failed. */
-        this._failures = []
+    /** @type {array} Failure information for each test that has failed. */
+    this._failures = []
+  }
 
-    }
-
-    /**
+  /**
      Assert that a condition is true.
      If the assertion fails, the failure is recorded for later reporting by the tester.
 
      @param {bool} condition The condition we're asserting.
      @param {string} description A description of the test.
      */
 
-    assert(condition, description) {
-        if (!condition) {
-            if (!description) description = ""
-            this._testFailures.push(description)
-        }
+  assert(condition, description) {
+    if (!condition) {
+      throw new Error(description || '')
```

Comment:
I also change the `assert` methods so that it throws and end the test instead of continuing the test and potentially crashing sketch when there are multiple asserts in a row (out of bounds index and stuff like this)